.gitignore

```diff
@@ -43,6 +43,7 @@
 # project-specific
 /node_modules
 /vendor/cache
+/vendor/bundle
```

Comment:
This folder was only present while in the CI, and was adding lots of undesired occurrences to our metrics. 

![image](https://github.com/cherrypush/cherry/assets/1740848/e1844d11-608a-4dfe-8148-9b5530914f88)